CHANGELOG.md

```diff
@@ -35,6 +35,10 @@
 
 ## 0.7.x
 
+### 0.7.5
+
+* Separate block positioner select data and make it a property of editor instead of block.
```

Comment:
Could reference the original issue here that details the problem, only because this doesn't explain why this change was made

Reply:
Good point.